dev.c

```diff
@@ -1302,7 +1302,7 @@ const struct file_operations fuse_dev_operations = {
 	.release	= fuse_dev_release,
 	.fasync		= fuse_dev_fasync,
 };
-#else
+#elif LINUX_VERSION_CODE <= KERNEL_VERSION(6,13,0)
 const struct file_operations fuse_dev_operations = {
 	.owner		= THIS_MODULE,
```

Comment:
#if LINUX_VERSION_CODE < KERNEL_VERSION(6,0,0)
/// we have the definition of no_llseek
    .llseek = no_llseek
#else
    /// pxd devices never supported seek operation.
    .llseek = NULL
#endif